server/src/main/java/com/netflix/conductor/Conductor.java

```diff
@@ -56,4 +59,23 @@ private static void loadExternalConfig() throws IOException {
             }
         }
     }
+    
+    private static void loadExternalConfigFromArgs(String[] args) throws IOException {
```

Comment:
Is it possible to provide the configs as a file and leverage `CONDUCTOR_CONFIG_FILE`? Two ways to accomplish the same outcome usually adds confusion, we also have to consider the possibility of `CONDUCTOR_CONFIG_FILE` and command line args present at the same time.